src/styles.typ

```diff
@@ -23,21 +57,33 @@
   if root != none {
```

Comment:
Is this loop necssary if `inherit` exists? Or could the logic in `inherit` be put here instead?